tests/tests_common_test.go

```diff
@@ -204,3 +208,27 @@ func getResourceKey(obj controllerutil.Object) client.ObjectKey {
 		Name:      obj.GetName(),
 	}
 }
+
+func expectUserCan(user string, groups []string, verb, resNamespace, resGroup, resVersion, resKind, resName, resSubresource string) {
+	sar, err := coreClient.AuthorizationV1().SubjectAccessReviews().Create(ctx, &authv1.SubjectAccessReview{
+		Spec: authv1.SubjectAccessReviewSpec{
+			User:   user,
+			Groups: groups,
+			ResourceAttributes: &authv1.ResourceAttributes{
+				Namespace:   resNamespace,
+				Verb:        verb,
+				Group:       resGroup,
+				Version:     resVersion,
+				Resource:    resKind,
+				Subresource: resSubresource,
+				Name:        resName,
+			},
+		},
+	}, metav1.CreateOptions{})
+	ExpectWithOffset(1, err).ToNot(HaveOccurred())
+	ExpectWithOffset(1, sar.Status.Reason).ToNot(BeEmpty(),
```

Comment:
I don't think this is an correct assertion
Reason field can be and sometimes is empty even if the Status is allowed

Reply:
Oh, good catch, I wanted to remove that and forgot, will remove now

Reply:
Done